app/javascript/pages/ReservationList.js

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import NavBar from "../components/Navbar";
+import { useSelector } from "react-redux";
+import getHotelDetails from "../Redux/DetailsAction";
+
+
+function ReservationList() {
+  const user = useSelector((state) => state.user);
+  const user_id = user.id;
+  const [reservations, setReservations] = useState([]);
+
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch(`/api/v1/users/${user.id}/reservations/`);
+        const json = await response.json();
+        setReservations(json.data);
+      } catch (error) {
+        console.log("error", error);
```

Comment:
- [ ] Please, kindly remove this old piece of debugging code to improve the cleanliness of the repo 👍